### ISSUE TITLE
Adyen 2.4.5 NullPointer in CheckoutSessionActivity.java:58

### ISSUE THREAD

**Issue by ovy9086** [Adyen/adyen-android#85]
Using :
```
implementation 'com.adyen.checkout:core:2.4.5'
implementation 'com.adyen.checkout:ui:2.4.5'
```
We get random crashes from LIVE with the following stacktrace :

```
java.lang.RuntimeException Unable to start activity ComponentInfo{com.rituals.app.prod/com.adyen.checkout.ui.internal.card.CupSecurePlusOneClickDetailsActivity}: java.lang.NullPointerException: Attempt to invoke interface method 'com.adyen.checkout.core.PaymentHandler com.adyen.checkout.core.PaymentReference.getPaymentHandler(android.app.Activity)' on a null object reference 
    ActivityThread.java:2665 android.app.ActivityThread.performLaunchActivity
    ActivityThread.java:2726 android.app.ActivityThread.handleLaunchActivity
    ActivityThread.java:-1 android.app.ActivityThread.-wrap12
    ActivityThread.java:1477 android.app.ActivityThread$H.handleMessage
    Handler.java:102 android.os.Handler.dispatchMessage
    Looper.java:154 android.os.Looper.loop
    ActivityThread.java:6119 android.app.ActivityThread.main
    Method.java:-2 java.lang.reflect.Method.invoke
    ZygoteInit.java:886 com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run
    ZygoteInit.java:776 com.android.internal.os.ZygoteInit.main
    XposedBridge.java:107 de.robv.android.xposed.XposedBridge.main


Caused by: java.lang.NullPointerException Attempt to invoke interface method 'com.adyen.checkout.core.PaymentHandler com.adyen.checkout.core.PaymentReference.getPaymentHandler(android.app.Activity)' on a null object reference 
    CheckoutSessionActivity.java:58 com.adyen.checkout.ui.internal.common.activity.CheckoutSessionActivity.onCreate
    CupSecurePlusOneClickDetailsActivity.java:84 com.adyen.checkout.ui.internal.card.CupSecurePlusOneClickDetailsActivity.onCreate
    Activity.java:6679 android.app.Activity.performCreate
    Instrumentation.java:1118 android.app.Instrumentation.callActivityOnCreate
    ActivityThread.java:2618 android.app.ActivityThread.performLaunchActivity
    ActivityThread.java:2726 android.app.ActivityThread.handleLaunchActivity
    ActivityThread.java:-1 android.app.ActivityThread.-wrap12
    ActivityThread.java:1477 android.app.ActivityThread$H.handleMessage
    Handler.java:102 android.os.Handler.dispatchMessage
    Looper.java:154 android.os.Looper.loop
    ActivityThread.java:6119 android.app.ActivityThread.main
    Method.java:-2 java.lang.reflect.Method.invoke
    ZygoteInit.java:886 com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run
    ZygoteInit.java:776 com.android.internal.os.ZygoteInit.main
    XposedBridge.java:107 de.robv.android.xposed.XposedBridge.main
```

**Comment by caiofaustino** [comment]
Ok so, if you are using `implementation 'com.adyen.checkout:ui:2.4.5'` you don't need to use `implementation 'com.adyen.checkout:core:2.4.5'` as it is already a sub dependency.

It looks like the Activity is missing a parameter from the Intent that started it. Are you trying to start the activity directly?

**Comment by ovy9086** [comment]
No, we do not start the Activity directly.

**Comment by caiofaustino** [comment]
We just released version `2.4.9` with a fix. Please update and see if it's solved. Feel free to reopen the issued if it still persists.
